README.md

```diff
@@ -27,28 +27,43 @@ Other requirements:
 
 ## Table of contents
 
-- [Installation & Configuration](#installation)
-  - [Using with schema.rb](#using-with-schemarb)
-  - [Configuring models](#configuring-models)
-  - [Backfill data](#backfill-data)
-  - [Log size limits](#log-size-limits)
-  - [Tracking only selected columns](#tracking-only-selected-columns)
-  - [Logs timestamps](#logs-timestamps)
-- [Usage](#usage)
-  - [Basic API](#basic-api)
-  - [Track meta information](#track-meta-information)
-  - [Track responsibility](#track-responsibility)
-  - [Disable logging temporary](#disable-logging-temporary)
-  - [Reset log](#reset-log)
-  - [Creating full snapshot instead of diffs](#full-snapshots)
-  - [Associations versioning](#associations-versioning)
-- [Dealing with large logs](#dealing-with-large-logs)
-- [Handling records deletion](#handling-records-deletion)
-- [Handling PG exceptions](#handling-pg-exceptions)
-- [Upgrading](#upgrading)
-- [Log format](#log-format)
-- [Troubleshooting 🚨](#troubleshooting)
-- [Development](#development)
+- [Logidze](#logidze)
```

Comment:
Why do we need this change?

Reply:
We don't -- I actually didn't intend to make it, but I think my markdown auto-formatter in my editor did it. I will revert!